test/utils/browser.js

```diff
@@ -2,22 +2,24 @@
 var assert = require('assert');
 var fs = require('fs');
 var functionToString = require('function-to-string');
-var async = require('async');
 var wd = require('wd');
 
-// Resolve cookies with helpful messaging
-var cookiesJson;
-try {
-  cookiesJson = fs.readFileSync(__dirname + '/../cookies.json');
-} catch (err) {
-  throw new Error('Could not read `test/cookies.json`. Please make sure it exists. ' +
-      'If it doesn\'t, follow the steps in https://github.com/twolfson/google-music.js#testing');
-}
-var cookies = JSON.parse(cookiesJson);
-
 // Resolve the compiled script
 var script = fs.readFileSync(__dirname + '/../../dist/google-music.js', 'utf8');
 
+// Google Login Info you have to fill this in
+var username = process.env.GPM_USER;
+var password = process.env.GPM_PWD;
+
+// These are the BrowserStack username and key vars that should NEVER be
+// pushed to remote.  They are stored in secure env vars on travis.ci
```

Comment:
This comment is a bit awkwardly phrased -- it's not consistent with the commenting format of the rest of this script. How about something like

```js
// Extract BrowserStack credentials from environment variables
// DEV: These are stored securely in Travis CI. Please never ever commit/push them
```